integreat_cms/cms/models/pages/page.py

```diff
@@ -256,7 +256,7 @@ def archived(self) -> bool:
         """
         return self.explicitly_archived or self.implicitly_archived
 
-    def get_non_archived_children(self) -> Iterator[Page]:
+    def get_non_archived_children(self, include_parent: bool = True) -> Iterator[Page]:
```

Comment:
```suggestion
    def get_non_archived_children(self) -> Iterator[Page]:
```
Could you also revert this change that is no longer needed?